src/main/webapp/itempagescript.js

```diff
@@ -2,18 +2,24 @@
  * Retrieves ItemPageData and forms page using other functions.
  */
 async function loadItemPage() {
-  const itemId = getUrlParam('itemId');
+  $(document).ready(function() {
+    $('#navbar').load('navbar.html', function() {
+      $('#navbarDashboardSection').addClass('d-none');
```

Comment:
I think it would be better to make default the #navbarDashboardSection as hidden (in the html). And then you can make it only visible on the dashboard page

Reply:
Yeah I made the navbar dashboard section hidden by default in last commit.